bind_exporter.go

```diff
@@ -14,8 +14,8 @@ import (
 
 	"github.com/digitalocean/bind_exporter/bind"
 	"github.com/digitalocean/bind_exporter/bind/auto"
-	"github.com/digitalocean/bind_exporter/bind/v2"
-	"github.com/digitalocean/bind_exporter/bind/v3"
+	v2 "github.com/digitalocean/bind_exporter/bind/v2"
```

Comment:
No need for the import aliases here.

Reply:
👍VSCode seem to want to stick them in (not sure why), I've removed as requested